application/modules/reports/views/scripts/distribution/finalize.phtml

```diff
@@ -197,92 +254,131 @@ if (count($this->shipment) > 0) { ?>
 <br />
 <br />
 <br />
-<?php
-    if ($adminSession->is_ptcc_coordinator == 0 && isset($this->shipment[0]['status']) && $this->shipment[0]['status'] != 'finalized') { ?>
```

Comment:
Why did you take out `...&& $this->shipment[0]['status'] != 'finalized'` from here?

Reply:
Hmmm... that might be an oversight. It looks like I pasted it from here: https://github.com/SystemOneId/ept/blob/5da42ec7392527345813ee0d46d758f9a813f33d/application/modules/reports/views/scripts/distribution/shipment.phtml#LL271C38-L271C38 but it was a while ago now, so I can't recall. Let me know if I should hotfix the statement and commit it into the next deployment

Reply:
@prof83 yes I think this is a required control as no user, not even an admin should be able to regenerate reports for a finalized panel

Reply:
Ok ill quickly patch it in prod and include it in the next commit, thanks @brichardssa